crates/biome_css_parser/tests/css_test_suite/ok/property/property_border.css.snap

```diff
@@ -0,0 +1,1216 @@
+                                    keyword: OUTSET_KW@607..614 "outset" [] [Whitespace(" ")],
+                                },
+                                color: CssColor {
+                                    hash_token: HASH@614..615 "#" [] [],
+                                    value_token: CSS_COLOR_LITERAL@615..618 "f33" [] [],
+                                },
+                            },
+                        },
+                        important: missing (optional),
+                    },
+                    SEMICOLON@618..619 ";" [] [],
+                    CssDeclaration {
+                        property: CssBorderProperty {
+                            name: CssIdentifier {
+                                value_token: IDENT@619..630 "border" [Newline("\n"), Whitespace("    ")] [],
+                            },
+                            colon_token: COLON@630..632 ":" [] [Whitespace(" ")],
+                            value: CssBorder {
+                                line_width: CssLineWidthKeyword {
+                                    keyword: MEDIUM_KW@632..639 "medium" [] [Whitespace(" ")],
+                                },
+                                line_style: CssLineStyle {
+                                    keyword: DASHED_KW@639..646 "dashed" [] [Whitespace(" ")],
+                                },
+                                color: CssColor {
+                                    hash_token: HASH@646..647 "#" [] [],
+                                    value_token: CSS_COLOR_LITERAL@647..650 "000" [] [],
+                                },
+                            },
+                        },
+                        important: missing (optional),
+                    },
+                    SEMICOLON@650..651 ";" [] [],
+                    CssDeclaration {
+                        property: CssBorderProperty {
+                            name: CssIdentifier {
+                                value_token: IDENT@651..662 "border" [Newline("\n"), Whitespace("    ")] [],
+                            },
+                            colon_token: COLON@662..664 ":" [] [Whitespace(" ")],
+                            value: CssBorder {
+                                line_width: CssRegularDimension {
+                                    value_token: CSS_NUMBER_LITERAL@677..679 "10" [] [],
+                                    unit_token: IDENT@679..681 "px" [] [],
+                                },
+                                line_style: CssLineStyle {
+                                    keyword: GROOVE_KW@664..672 "groove" [] [Whitespace("  ")],
+                                },
+                                color: CssColor {
+                                    hash_token: HASH@672..673 "#" [] [],
+                                    value_token: CSS_COLOR_LITERAL@673..677 "fff" [] [Whitespace(" ")],
+                                },
```

Comment:
Look mom! The AST can track these in any order!

Reply:
Let's GOOOOO!